examples/flutter_gallery/lib/demo/cupertino/cupertino_dialog_demo.dart

```diff
@@ -73,10 +73,12 @@ class _CupertinoDialogDemoState extends State<CupertinoDialogDemo> {
               showDemoDialog<String>(
                 context: context,
                 child: new CupertinoAlertDialog(
-                  title: const Text('Allow "Maps" to access your location while you use the app?'),
+                  title: const Text('Allow "Maps" to access your location while'
+                      ' you use the app?'),
```

Comment:
Please put the space at the end of the first line, not the second. Also, put a newline before the first. As in:

```dart
                 title: const Text(
                   'Allow "Maps" to access your location while '
                   'you use the app?',
                 ),
```
￼...though once you do that, maybe it'll just fit on one line anyway.

Reply:
doesn't continuations indent another two spaces on the second line from the first line?

Reply:
There's various ways to do it, but we try to optimize for readability, so that argues for keeping the strings indented the same amount. See also https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo